internal/checker/terraform/rules/ibm/classic_infrastructure/tfibm_timeout_configuration_is_set.rego

```diff
@@ -0,0 +1,63 @@
+# METADATA
+# title: "Verify timeout configuration is set"
+# description: "This policy ensures that a valid timeout configuration is established. It verifies that a timeout value for the creation process is specified, enhancing the reliability and efficiency of the system."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/lb
+# custom:
+#   id: CB_TFIBM_059
+#   severity: MEDIUM
+package lib.terraform.CB_TFIBM_059
+
+import rego.v1
+
+isvalid(block) if {
+	block.Type == "resource"
+	some label in block.Labels
+	supported_resource := ["ibm_lb", "test_lb_local"]
+	label in supported_resource
+}
+
+resource contains resource if {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource contains resource if {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+offload(attribute) if {
+	attribute.ssl_offload == true
+}
+
+pass contains resource if {
```

Comment:
[resource](https://registry.terraform.io/providers/IBM-Cloud/ibm/latest/docs/resources/lb#ssl_offload)

``ssl_offload - (Optional, Bool)``

attribute is optional, it should be on a fail block, since it is not mandated to be available in the terraform file. Update test script after changing the policy rules.